bin/git_utils.py

```diff
@@ -0,0 +1,118 @@
+import git
+import os
+import json
+import logging
+
+
+def get_git_repo():
+    """
+    Create a git.Repo object for this repository
+
+    Returns
+    -------
+    git.Repo
+        Repo object for the current git repository
+    """
+    return git.Repo(os.getcwd(), search_parent_directories=True)
+
+
+def create_git_ml(repo):
```

Comment:
Can we rename this function to 'create_git_cml'? Where C in cml stands for collaborative/continuous/communal.

Similarly can we refactor other functions/variables/folder paths/doc strings that have git_ml to git_cml. I made most of these changes in the previous pull request that I just merged with main, but since we refactored the code, I don't know if just resolving the merge conflicts will fix this.